wscript

```diff
@@ -253,6 +267,16 @@ def configure(conf):
 		conf.env.append_unique('LINKFLAGS_cshlib', ['-nostdlib', '-nostartfiles'])
 		conf.env.append_unique('LINKFLAGS_cxxshlib', ['-nostdlib', '-nostartfiles'])
 
+	# same on the vita
+	if conf.env.DEST_OS == 'psvita':
+		conf.env.append_unique('CFLAGS_cshlib', ['-fPIC'])
+		conf.env.append_unique('CXXFLAGS_cxxshlib', ['-fPIC', '-fno-use-cxa-atexit'])
+		conf.env.append_unique('LINKFLAGS_cshlib', ['-nostdlib', '-Wl,--unresolved-symbols=ignore-all'])
+		conf.env.append_unique('LINKFLAGS_cxxshlib', ['-nostdlib', '-Wl,--unresolved-symbols=ignore-all'])
+		# set the metadata
+		conf.env.TITLEID = 'XASH10000'
+		conf.env.APPNAME = 'xash3d-fwgs'
```

Comment:
Can this be passed through main executable build arguments, like `nacp` or `icon` paths on NSwitch?

Reply:
Done in b6c8fe3.